src/features/WaitingRoom.js

```diff
@@ -32,89 +30,131 @@ useEffect(() => {
-        </ListItem>
-          ))}
-        </List>
-      </Box>
+  if (show && playerID) {
+    return navigate(`/game/apples/${localStorage.getItem('matchID')}`);
+  } else {
+    return (
       <Box
-       sx={{
-        display:"flex",
-        maxWidth: 500,
-        p:'1rem',
-        ml: {md: '2rem'},
-        mt: '0.5rem'
-      }}
+        sx={{
+          display: 'flex',
+          maxWidth: 500,
+          height: '80vh',
+          flexDirection: 'column',
+          alignItems: 'center',
+          justifyContent: 'center',
+          p: '1rem',
+          ml: { md: '2rem' },
+          mt: '0.5rem',
+          zIndex: 1,
+        }}
       >
-        {/* {playerID == 0 ?
-        <Button variant="contained"
-          // disabled={}
+        <Typography
+          variant="h4"
           sx={{
-            p:'0.5rem',
-            width: '50%',
-            ml: '1rem',
-            fontSize: {sm: '1rem', md: '1.wrem'}
+            fontSize: { xs: '1.7rem', sm: '2rem', md: '2.2rem' },
+            mt: '0.6rem',
+          }}
+        >
+          Waiting for players
+        </Typography>
+        <Box
+          sx={{
+            textAlign: 'center',
+            background: 'blue',
+            p: '0.5rem',
+            mt: '1rem',
+            fontSize: { sm: '1rem', md: '1.2rem' },
+          }}
+        >
+          {matchID}
+        </Box>
+        <Box>
+          <List
+            sx={{
+              width: '100%',
+              maxHeight: 400,
+              bgcolor: 'color',
+              p: '1rem',
+              overflowY: 'scroll',
             }}
-            onClick={() => setShow(true)}>Start Game</Button> : null} */}
-          <Button variant="contained"
+          >
+            {players.map((value, index) => (
+              <ListItem key={value + index} disableGutters>
+                <ListItemText
+                  primaryTypographyProps={{
+                    fontSize: { sm: '18px', md: '20px' },
+                  }}
+                  primary={`${index + 1}. ${
+                    value.name === undefined ? '' : value.name
+                  }`}
+                />
+              </ListItem>
+            ))}
+          </List>
+        </Box>
+        <Box
+          sx={{
+            display: 'flex',
+            maxWidth: 500,
+            p: '1rem',
+            ml: { md: '2rem' },
+            mt: '0.5rem',
+          }}
+        >
+          {playerID == 0 ? (
+            <Button
+              variant="contained"
+              // disabled={}
+              sx={{
+                p: '0.5rem',
+                width: '50%',
+                ml: '1rem',
+                fontSize: { sm: '1rem', md: '1.wrem' },
+                outline: '1px solid white',
+              }}
+              onClick={() => setShow(true)}
+            >
+              Start Game
```

Comment:
are we adding this back in?

Reply:
No. I added it unintentionally. Kesang is going to revert the change when he merges his part.